js/maps-setup.js

```diff
@@ -55,26 +54,93 @@ function initializeMap() {
                                  my_map_options);
     // this is an *array* that holds all the marker info
     var all_my_markers =
-            [{position: new google.maps.LatLng(41.9000,12.5000),
-              map: my_map,
-              icon: blueURL, // this sets the image that represents the marker in the map to the one
-                             // located at the URL which is given by the variable blueURL, see above
-              title: "first Marker",
-              window_content: "<h1>Marker1</h1><p> and this would be the extended description</p>"
-             },
-             {position: new google.maps.LatLng(41.8902,12.4923),
-              map: my_map,
-              icon: blueURL, // this sets the image that represents the marker in the map
-              title: "second Marker",
-              window_content: "<h1>Marker2</h1><p> and <a href='http://something'>this would</a> be the extended description</p>"
-            },
-            {position: new google.maps.LatLng(41.8986,12.4768),
-             map: my_map,
-             icon: redURL, // this sets the image that represents the marker in the map
-             title: "third Marker",
-             window_content: '<h1>Marker3</h1><img title="Picture of Quote. Src: someone, some year"  src="https://s-media-cache-ak0.pinimg.com/736x/6d/e2/25/6de2251b8b4be709dcc936ae4f0caaaf.jpg"/>' +
-             '<blockquote>quote quote quote quote</blockquote>'
-           }
+            [
+              {position: new google.maps.LatLng(34.393638, 132.464921),
```

Comment:
I'd suggest adding pictures of these places to the infowindow text.  Some additional info about each place would also really help.